### ISSUE TITLE
Disable indexing of ListProperty()s does not work atm

### ISSUE THREAD

**Issue by mkustermann** [dart-lang/appengine#66]
The error returned by the datastore service is:

```
ApplicationError: Exclude from indexes cannot be set on a list value
```

Most likely the "do not index" bit must be set on the individual list items - instead of the entire list - in the new API.

**Comment by mkustermann** [comment]
A [CL](https://codereview.chromium.org/2857723004) is up for review.

**Comment by enyo** [comment]
The CL fixed the issue. Thank you so much for providing a fix and releasing the new version this quickly!

**Comment by mkustermann** [comment]
You are welcome!

(fixed in 511ebc473dad886dac4c19978d04635e30e3623c)
